src/EditorFeatures/Test2/IntelliSense/CSharpCompletionCommandHandlerTests.vb

```diff
@@ -6923,7 +6923,6 @@ namespace NS2
                 Dim document = state.Workspace.CurrentSolution.GetDocument(state.Workspace.Documents.Single(Function(d) d.Name = "C.cs").Id)
 
                 Dim completionService = document.GetLanguageService(Of CompletionService)()
-                completionService.GetTestAccessor().SuppressPartialSemantics()
```

Comment:
This test involves 2 documents, will using frozen partial cause flakiness? @jasonmalinowski

Reply:
@jasonmalinowski -- any thoughts here?

Reply:
ping @jasonmalinowski

Reply:
def seemsl ike it could.  what's the thinking on why this is ok to remove here?

Reply:
Don't remember exactly why I went down that path, but it doesn't seem necessary for this PR, and probably not a good thing as you both mentioned. Added back.